CONTRIBUTING.md

```diff
@@ -165,7 +165,7 @@ You can download Eclipse and load the Frank!Framework sources into it using the
 - Optionally (when you have access to the proprietary jars some modules depend on) add your Nexus credentials and enable the proprietary profile in your maven settings.xml
 - In the Git Perspective, right click the IAF Repository and click 'Import Projects...'
 - The Import wizard appears which allows you to import many different kinds of projects.
-- **deselect**: iaf-coolgen, iaf-ibm, iaf-ifsa, iaf-sap, iaf-tibco, iaf-idin, docker\* and ear (unless you have access to the proprietary repository), Finish.
+- **deselect**: iaf-ibm, iaf-ifsa, iaf-sap, iaf-tibco, iaf-idin, docker\* and ear (unless you have access to the proprietary repository), Finish.
```

Comment:
@nielsm5 Kloppen al deze modules in dit lijstje nog? Ik denk dat er nog wat meer weg mogen en misschien wat nieuwe erbij?

Reply:
Ehh, ik zie dat IBM ook weg kan ja

Reply:
Ik raak een beetje het overzicht kwijt. Ik dacht dat IDIN en IFSA ook al weg waren maar zie dat die er nog staan.

Reply:
Haha, tja, IDIN en IFSA staan nog op de shortlist, dus ik wil daar nog even mee wachten..

Reply:
```suggestion
- **deselect**: iaf-ifsa, iaf-sap, iaf-tibco, iaf-idin, docker\* and ear (unless you have access to the proprietary repository), Finish.
```